contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
```

Comment:
So the `instanceRef` section here is ok then... but since bindings are namespaces, I would still suggest adding a namespace to the `metadata` section since that would demonstrate a more realistic use. Same goes for the instance example.